repos/re-theme/src/hooks/usePointerState.js

```diff
@@ -133,10 +133,10 @@ const useElementEvents = (options=noOpObj, pointerState) => {
     // Use the Dom event names, NOT the react event names,
     // because these events are directly added to the element through the Dom API
     const events = pointerState === 'hover'
-      ? { mouseenter: onMouseIn, mouseleave: onMouseOut }
+      ? { pointerover: onMouseIn, pointerout: onMouseOut }
```

Comment:
Because we changed this to be `pointer` events, I think it would make sense to also change the corresponding method names.
So instead of `{ pointerover: onMouseIn, pointerout: onMouseOut }`
It would be `{ pointerover: onPointerIn, pointerout: onPointerOut }`

This goes for the passed in custom callbacks as well.

Reply:
updated names and tests